docs/website/scripts/live-blocks/scripts/inject

```diff
@@ -18,6 +18,4 @@ mkdir -p ../../public/live-blocks # Other assets
 cp -r static/* ../../public/live-blocks/
 
 # Preprocess the docs
-npm run --silent preprocess ../../public/index.html
-npm run --silent preprocess ../../public/docs/*/*.html
-npm run --silent preprocess ../../public/docs/*/*/*.html
\ No newline at end of file
+find ../../public/ -name '*.html' -print0 | xargs -0 -n200 npm run preprocess
```

Comment:
I'm curious if you think we would need to raise this value again or this is something we need to try out ?

Reply:
We could tweak it to make things slightly faster. As far as I am concerned, as long as it's faster than the go perf test, it's fast enough...